client-participation/gulpfile.js

```diff
@@ -108,7 +109,7 @@ function prepPathForTemplate(path) {
 gulp.task("connect", [], function () {
   function proxyToPreprod(req, response) {
     var x = request(
-      (polisConfig.SERVICE_URL || "https://preprod.pol.is") + req.originalUrl
+      (polis_config.get('service_url') || "https://preprod.pol.is") + req.originalUrl
```

Comment:
This certainly isn't a hold up on merging, but does convict allow us to specify defaults at a top level, so we don't have to hard code fallbacks like this? Would we actually want to do that?

Reply:
Convict allows to define defaults in for example the [schema.yaml](https://github.com/crkrenn/polis/blob/dev-for-ci-test/config/schema.yaml) file so it would be an easy change. Should I do it?

Reply:
Yes, please. After thinking about it a bit, I think it's better that any defaults like this be in the `schema.yaml` so that all defaults are in the same place.

Part of my question was wondering aloud whether we wanted _any_ default for this, but I think it makes sense to leave in for folks who want to run the clients against the main deployments.